FileAPI/url/resources/fetch-tests.js

```diff
@@ -0,0 +1,56 @@
+function fetch_tests(fetch_method, fetch_should_succeed, fetch_should_fail) {
+  const blob_contents = 'test blob contents';
+  const blob = new Blob([blob_contents]);
+
+  promise_test(t => {
+    const url = URL.createObjectURL(blob);
+
+    return fetch_should_succeed(t, url).then(text => {
```

Comment:
`fetch_method` seems to be displayed, but not used when issuing the actual fetch requests.